docs/04-operation-guides/troubleshooting/eventing/evnt-04-free-jetstream-storage.md

```diff
@@ -6,9 +6,10 @@ title: Eventing backend stopped receiving events due to full storage
 
 NATS JetStream backend stopped receiving events due to full storage.
 
-You see one of the following messages:
-- `507 Insufficient Storage` error from Event Publisher Proxy.
-- `no space left on device` from the Eventing backend.
+You observe the following behaviour in the Event Publisher Proxy(EPP):
+
+- `507 Insufficient Storage` HTTP Status from EPP on publish request.
```

Comment:
```suggestion
- `507 Insufficient Storage` HTTP Status from Event Publisher Proxy on publish request.
```

Reply:
Since I already introduce the EPP abbreviation above, I think it's valid to use `EPP` here.

Reply:
Well, its upto TWs to decide if its okay for public docs. :)

Reply:
Since the abbreviation was introduced before, it's ok to use it later in the doc :)